doc/content_overview.md

```diff
@@ -22,17 +22,16 @@ Implemented in [search_payload_presenter.rb](https://github.com/alphagov/publish
  - 20 places
  - 8 help pages
 
-
 ## Travel-advice-publisher
 Publishes [foreign travel advice](https://www.gov.uk/foreign-travel-advice) on GOV.UK. There are 226 pages, but they have
 a format of "custom application" in Rummager.
 
-Implemented in [search_payload_presenter.rb](https://github.com/alphagov/travel-advice-publisher/blob/master/app/presenters/search_payload_presenter.rb).
+Published via the publishing API
```

Comment:
We could probably just delete this page from the docs. I just wrote it so I could quickly compare what the apps were doing

Reply:
done